config/drupal/phpcsfixer.rules.yml

```
@@ -24,15 +24,16 @@ parameters:
                 - try
                 - while
                 - yield
-        braces:
-            allow_single_line_closure: true
-            position_after_functions_and_oop_constructs: same
-            position_after_control_structures: same
-            position_after_anonymous_constructs: same
+        curly_braces_position:
+            classes_opening_brace: same_line
+            control_structures_opening_brace: same_line
+            functions_opening_brace: same_line
+        compact_nullable_type_declaration: true
         comment_to_phpdoc: false
-        compact_nullable_typehint: true
         constant_case:
             case: upper
+        control_structure_continuation_position:
+            position: next_line
```

if previously was

```
       braces: position_after_control_structures: same
```

this should not be ?

```suggestion
        control_structure_continuation_position:
            position: same_line
```

---

Thinking about it .... I think using 

```
        control_structure_continuation_position:
            position: next_line
```

is correct and it replace the Fixer we made `Drupal/control_structure_braces_else: true`